software/control_sw/src/blocks/fpga.py

```diff
@@ -77,6 +77,20 @@ def get_build_time(self):
         t = self.read_uint('version_timestamp')
         return t
 
+    def set_connected_antname(self, antname):
+        """
+        Set the connected antenna name.
+        :param antname: The antenna name.
+        :type antname: str
+        """
+        self.antname = antname
+
+    def get_connected_antname(self):
```

Comment:
The docstring convention somewhat demands that you define what this function returns